libs/eslint-plugin/src/lib/rules/prefer-no-lodash-clone-deep.ts

```diff
@@ -0,0 +1,32 @@
+import { ESLintUtils, TSESTree } from '@typescript-eslint/experimental-utils';
+import * as path from 'path';
+import { docsUrl } from '../utils/docs';
+
+const MESSAGE_ID = 'no-lodash-clone-deep';
+export type MessageIds = typeof MESSAGE_ID;
+
+export default ESLintUtils.RuleCreator(docsUrl)({
+  name: path.parse(__filename).name,
+  meta: {
+    docs: {
+      recommended: 'error',
+      description: 'Detects all usages of cloneDeep from Lodash.',
+    },
+    type: 'problem',
+    messages: {
+      [MESSAGE_ID]:
+        'Use with caution - cloneDeep is a costly operation that may cause performance problems.',
```

Comment:
we could give a hint here and point to `structuredClone` https://web.dev/structured-clone/

Reply:
Good suggestion :+1: I've added it now.